docs/elastic_inference.md

```diff
@@ -0,0 +1,182 @@
+# Model Serving with Amazon Elastic Inference 
+
+## Contents of this Document
+* [Introduction](#introduction)
+* [Custom Service](#custom-service)
+* [Creating a EC2 instance with EIA support](#creating-a-ec2-instance-with-eia-support)
+* [Custom Service file with EIA](#custom-service-file-with-eia)
+* [Running elastic inference on a resnet-152](#running-elastic-inference-on-a-resnet-152)
+## Introduction
+
+Amazon Elastic Inference allows you to attach low-cost GPU-powered acceleration to Amazon EC2 and Amazon SageMaker instances to reduce the cost of running deep learning inference by up to 75%. With MMS it is easy to deploy an MXNet based model, taking advantage of the attacble hardware accelerator.
+In this document we explore using Elastic Inference Acclerator (EIA) attached to a Compute Optimized EC2 instance.
+
+## Custom Service
+
+The capability to run model inference with the Elastic Inference Accelerator (EIA) can be achieved by building the custom service to use the EIA context rather than a GPU or CPU context. An MXNet version with support for EIA is required.
+
+To understand the basics of writing a custom service file refer to the [Custom Service Documentation](https://github.com/awslabs/mxnet-model-server/blob/master/docs/custom_service.md).
+
+## Creating a EC2 instance with EIA support 
+
+To Create an EC2 instance with EIA support there are few pre-requisites. These include:
+1. Configuring a Secuirty Group
+2. Creating a secure Endpoint.
+3. Creating a IAM role policy.
+    
+The above steps are explored in detail in [AWS Elastic Inference documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-ei.html)  
+
+On completing the above steps, following steps need to be followed in launching a instance with EIA support
+
+1. Open the Amazon EC2 console at https://console.aws.amazon.com/ec2/.
+
+2. Choose Launch Instance.
+
+3. Under Choose one of the Deep Learning AMIs, we recommend Deep Learning AMI v20. This is required to use MXNet with EIA.
+
+4. Under Choose an Instance Type, we recommend a compute optimized EC2 instance such as c5.2xlarge.
+
+5. Choose Next: Configure Instance Details.
+
+6. Under Configure Instance Details, check the configuration settings. Ensure that you are using the VPC with the security groups for the instance and the Amazon EI accelerator that was setup earlier. For more information, see Configuring Your Security Groups for Amazon EI.
+
+7. For IAM role, select the role that you created in the Configuring an Instance Role with an Amazon EI Policy procedure explained in the above documentation.
+
+8. Select Add an Amazon EI accelerator.
+
+9. Select the size of the Amazon EI accelerator. Your options are: eia1.medium, eia1.large, and eia1.xlarge. 
```

Comment:
It might be a good idea to mention that certain models require a larger EIA. Not sure if there are guidelines for this yet.

Reply:
I am not sure of this either, However I will add a brief note on the same.